source/includes/_tipser-elements.md

```diff
@@ -102,6 +102,77 @@ All properties are optional:
 property name | description |type | default value
 --------------|-------------|-----|---------------
 defaultAddedToCartPopup | controls whether default Added To Cart Popup is displayed | boolean | true 
+eventsHandlers | gives possibility to add handlers to Tipser Elements events | object | empty
+useDefaultErrorHandler | if set to false there will be no error message displayed in case of error | boolean | true
+
+#### Event Handlers
+Event handlers may be passed as a part of `config` option of `TipserElementsProvider`. There is a number of events exposed for developer. Each of the handler may have a different params as arguments of handler. 
+
+```js
+
+let tipserConfig = {
+    lang: 'en',
+    primaryColor: 'red',
+    // ----- EVENT HANDLING START
+    useDefaultErrorHandler: true,
+    eventsHandlers: {
+        onError: (error) => {
+            console.log(error)
+        },
+        onAddToCart: ({cartSize, product}) => {
+            console.log('Hurray, you have added item to cart. ', product);
+            console.log('Your cart size is now. ', cartSize);
+        }
+    }
+    // -- EVENT HANDLING END
+};
+
+class App extends Component {
```

Comment:
I believe we don't really need to provide e2e example how to pass configuration to `TipserElementsProvider` as this is done earlier, here: https://github.com/Tipser/docs/blob/604337c3b641201382ce82213e6f931499f797c0/source/includes/_tipser-elements.md#example-of-above-components-used-together. 
I think creation of configuration object would be enough.

Reply:
I changed to // ... as in example above class App extends Component { ... } WDYT ?